### PR TITLE
AOSL-18 : Add expose filter for Advanced Search

### PR DESCRIPTION
**Pull request by monishdeb**
This PR include following items
1. CSS and JS fixes for advanced seach form
2. Add camp session date filters, proximilty address filters, 'Accepting Clients?' boolean filter, 'Services are provided' and 'Age groups served' checklist filter

### CODE REVIEW

File: jma_customizations.module [hunk 1]
```diff
@@ -552,3 +555,71 @@ function jma_customizations_views_data_alter(&$data) {
...
+  }
+  else {
+    $request_url = GoogleMaps::$GOOGLEMAPSAPIURLBASE;
+  }
+  $request_url .= '/maps/api/geocode/json?address=' . urlencode($address);
+
+  if (!empty($config->get('google_map_api_server_key'))) {
+    $request_url .= '&key=' . $config->get('google_map_api_server_key');
+  }
+  elseif (!empty($config->get('google_map_api_key'))) {
```
Comment (seamuslee001) [inline jma_customizations.module:704]: @monishdeb isn't this just redundant / silly

File: js/google-map.js [hunk 2]
```diff
@@ -1,9 +1,75 @@
...
+      var $currentlocation = $('#edit-current-location');
+
+      // Get the geolocation from the browser.
+      navigator.geolocation.getCurrentPosition(
+
+        // Success handler for getCurrentPosition()
+        function (position) {
+          var lat = position.coords.latitude;
+          var lng = position.coords.longitude;
+          var accuracy = position.coords.accuracy / 1000;
```
Comment (seamuslee001) [inline js/google-map.js:64]: Is this going to be used for anything?

File: jma_customizations.module (hunk 1 above)
Comment (monishdeb) [reply jma_customizations.module:704]: umm there is a difference, as it's fetching the key from google_map_api_server_key or google_map_api_key setting. In our setup, later is used, but always good to have flexible code isn't it!!

File: js/google-map.js (hunk 2 above)
Comment (monishdeb) [reply js/google-map.js:64]: oops thanks for noticing it, will remove.

File: jma_customizations.module (hunk 1 above)
Comment (seamuslee001) [reply jma_customizations.module:704]: oh i didn't spot the word server, why is there a diffence? module versions?

**Pull request merged**